### PR TITLE
Fixing a bug with 'ControlledGate() does not take any arguments'

### PR DESCRIPTION
**Pull request by MSRudolph**
This bug occured when transforming a complicated z-quantum-core circuit with `export_to_qiskit`.

### CODE REVIEW

**Review by mstechly — COMMENTED** [review]
Would be nice to add a testcase for this, do you have a small example to re-create it?

**Comment by MSRudolph** [comment]
> Would be nice to add a testcase for this, do you have a small example to re-create it?

`from qeqiskit.conversions import export_to_qiskit`
That does it for me.
`qiskit` version `0.28.0`

**Review by AthenaCaesura — COMMENTED** [review]
Maybe I'm missing something, but it looks like you set the CSWAP to the same key as CRX. This would cause CSWAP to be left out of the dictionary. Just changing the key to what it was before `qiskit.circuit.library.CSwapGate` should fix it.

File: src/python/qeqiskit/conversions/_circuit_conversions.py [hunk 1]
```diff
@@ -84,7 +84,9 @@ def _factory(*gate_params):
 
 QISKIT_ZQUANTUM_GATE_MAP = {
     **{q_cls: z_ref for z_ref, q_cls in ZQUANTUM_QISKIT_GATE_MAP.items()},
-    qiskit.circuit.library.CSwapGate: _builtin_gates.SWAP.controlled(1),
+    qiskit.circuit.library.CRXGate: _make_controlled_gate_prototype(
```
[the remark this answers is not in the document]
Comment (MSRudolph) [reply src/python/qeqiskit/conversions/_circuit_conversions.py]: Oups, true! I must have made that mistake while copy-pasting. See new commit.

**Comment by MSRudolph** [comment]
Now it fails again. Unfortunately, I currently don't have the ability to search the bug.

**Comment by max-radin** [comment]
Just a note that the tests are failing because of Qiskit's changes to the statevector simulator, which will be fixed in ZQS-898.

**Pull request closed without merging**